Automattic/jetpack
# Support of tier type

Fixes https://github.com/Automattic/gold/issues/192

## Proposed changes:

* Support "tier" type for NL

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply D126468-code on your sandbox
* Using https://github.com/Automattic/wp-calypso/pull/83433, create a tier plan on any site (JN, wpcom simple)
* Make sure the new plans were created with type `tier`  in wpsh
```
$only_tiers = true;
print_r(Memberships_Product::get_product_list(<your blog id>, null, null. $only_tiers);
```
* if you have an existing tier, edit it and make sure the type is modified
* create a paid-newsletter post with this new plan
* make sure you can subscribe as before
* optional: make sure you can subscribe also with previously created tiers
* Repeat with WPCOM simple if you tested with JN
* Repeat with JN if you tested with WPCOM simple
